custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE {new_table} B
+SET
+B.pse_daily_attendance = A.pse_days_attended,
+B.pse_daily_attendance_male = CASE WHEN B.sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+B.pse_daily_attendance_female = CASE WHEN B.sex = 'F' THEN A.pse_days_attended ELSE NULL END
+FROM {old_table} A
+WHERE A.doc_id = B.doc_id and A.month_start = B.month_start
```

Comment:
We can likely do this tomorrow if you want.

I think @gcapalbo was asking if it is indexed as (doc.id, month_start), but the primary key is (doc.id, repeat_iteration) There is not a separate index on month_start. It may be worth adding a unique index on (doc.id, month_start) on both tables.

@sheelio would probably know this better, but I believe that (doc_id, month_start) is unique even though its not hte primary key